client/src/components/ToDoListItem/ToDoListItem.js

```diff
@@ -42,6 +42,7 @@ const CompletedButton = styled.button`
 
 const TitelSpan = styled.span`
   align-self: end;
+  justify-self: start;
```

Comment:
Does this have any effect? When I remove it in the inspector the title remains in place.

Reply:
Thanks, but when I remove it, my title moves to the center :)